sdks/go/pkg/beam/core/sdf/lock.go

```diff
@@ -80,3 +80,15 @@ func (rt *LockRTracker) GetRestriction() interface{} {
 	defer rt.Mu.Unlock()
 	return rt.Rt.GetRestriction()
 }
+
+// IsBounded locks a mutex for thread safety, and then delegates to the
+// underlying tracker's IsBounded(). If BoundableRTracker is not implemented
+// then the RTracker is considered to be bounded by default.
+func IsBounded(rt *LockRTracker) bool {
```

Comment:
Why is this an unused function instead of a method on *LockRTracker?

Reply:
Sorry, bad copying